Cargo.toml

```diff
@@ -1,9 +1,14 @@
 [package]
-name = "hilow-social-feed-api"
+name = "hilow"
```

Comment:
Improves simplicity for `use` statements.

So instead of having to do:

```rust
use hilow_social_feed_api;
```

We would do instead:

```rust
use hilow;
```